app/styles/app.scss

```diff
@@ -46,6 +46,16 @@ $purple: rgb(167, 41, 235);
   .navbar-nav {
     float: right;
   }
+  #footer {
+  padding-top: 50px;
+  text-align: center;
+  a{
+    color:black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Colon after property should be followed by one space